core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -237,26 +235,24 @@ public int getViewId() {
 
    @Override
    public void waitForView(int viewId) throws InterruptedException {
-      waitForView(viewId, 0, TimeUnit.SECONDS);
+      waitForView(viewId, Long.MAX_VALUE, TimeUnit.NANOSECONDS);
```

Comment:
How much easier? If I used seconds, I couldn't use `Long.MAX_VALUE`, because it would be transformed into a negative number of nanoseconds.